db/schema.rb

```diff
@@ -121,9 +140,8 @@
 
   add_foreign_key "activities", "activities"
   add_foreign_key "activities", "organisations", column: "extending_organisation_id"
-  add_foreign_key "activities", "organisations", column: "reporting_organisation_id"
```

Comment:
Odd that this unrelated bit was removed, oddness with database state during a rebase?